src/reanimated2/hook/useAnimatedGestureHandler.ts

```diff
@@ -31,78 +17,108 @@ const EventType = {
   END: 5,
 };
 
-interface GestureHandlerNativeEvent {
-  handlerTag: number;
-  numberOfPointers: number;
-  state: (typeof EventType)[keyof typeof EventType];
-}
+type StateType = (typeof EVENT_TYPE)[keyof typeof EVENT_TYPE];
 
-export interface GestureHandlerEvent<T> extends __NativeEvent<T> {
-  nativeEvent: T;
+interface PropsUsedInUseAnimatedGestureHandler {
```

Comment:
Could we have something more concise here?

Reply:
I could do `GestureHandlerEventPropsUsedInUseAnimatedGestureHandler` 🤔

Reply:
I don't want to add reliance on RNGH types here since it's deprecated API and we don't want to worry about this part of code breaking.